input/pagecontent/guidance_treatment.md

```diff
@@ -27,6 +28,9 @@ The `text` may be either:
 
 #### Ingredients
 
+Implementers SHOULD specify the ingredients of the medication.
+The absence of ingredients must be exceptional and restricted to borderline cases.
+
```

Comment:
Maybe it would be good to also add that ingredients that are active principles shall be flagged with `isActive` as `true`.

Reply:
Done!